### PR TITLE
修复learnku  descript 显示不正确和不加载内容的问题

### PR DESCRIPTION
**Pull request by kayw-geek**
## 完整路由地址 / Example for the proposed route(s)

```routes
/learnku/php
/learnku/laravel
```

## 说明 / Note
修复描述与标题一样的问题，修复无法加载内容的问题

### CODE REVIEW

File: lib/v2/learnku/maintainer.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+module.exports = {
+    '/learnku/:community/:category?': ['kayw-geek'],
```
Comment (TonyRL) [inline lib/v2/learnku/maintainer.js:2]:
```suggestion
    '/:community/:category?': ['kayw-geek'],
```

File: lib/v2/learnku/topic.js [hunk 2]
```diff
@@ -34,18 +34,25 @@ module.exports = async (ctx) => {
...
+                const itemData = itemResponse.data;
+                const article = cheerio.load(itemData);
                 return {
                     title: $('.topic-title').text().trim(),
+                    description: article('.article-content .content-body p').text().substring(0, 70) + '...',
                     category: categoryName,
-                    link: $('.topic-title-wrap').attr('href'),
-                    pubDate: new Date($('.timeago').attr('title')).toUTCString(),
+                    link: itemLink,
                 };
```
Comment (TonyRL) [inline lib/v2/learnku/topic.js:56]: Please add cache.
docs: https://docs.rsshub.app/joinus/quick-start.html#ti-jiao-xin-de-rsshub-gui-ze-bian-xie-jiao-ben-shi-yong-huan-cun

File: lib/v2/learnku/topic.js [hunk 3]
```diff
@@ -34,18 +34,25 @@ module.exports = async (ctx) => {
...
+                const itemLink = $('.topic-title-wrap').attr('href');
+                const itemResponse = await got({
+                    method: 'get',
+                    url: itemLink,
+                });
+                const itemData = itemResponse.data;
+                const article = cheerio.load(itemData);
                 return {
                     title: $('.topic-title').text().trim(),
+                    description: article('.article-content .content-body p').text().substring(0, 70) + '...',
```
Comment (TonyRL) [inline lib/v2/learnku/topic.js:53]: You can provide a fulltext description and let user use the [build-in brief parameter](https://docs.rsshub.app/parameter.html#shu-chu-jian-xun) to trim it down.

**Review by TonyRL — CHANGES_REQUESTED** [review]

File: lib/v2/learnku/topic.js (hunk 3 above)
Comment (kayw-geek) [reply lib/v2/learnku/topic.js]: okay

File: lib/v2/learnku/topic.js [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+
+            return {
+                title,
+                description: art(path.join(__dirname, 'templates/topic.art'), {
+                    article,
+                    comment,
+                }),
+                category: categoryName,
+                link: itemLink,
+                pubDate: new Date($('.timeago').attr('title')),
```
Comment (TonyRL) [inline lib/v2/learnku/topic.js:50]: Please use `parseDate()`.
Docs: https://docs.rsshub.app/joinus/pub-date.html

**Review by TonyRL — CHANGES_REQUESTED** [review]

File: lib/v2/learnku/topic.js (hunk 4 above)
Comment (kayw-geek) [reply lib/v2/learnku/topic.js]: Thank you for your review, This problem has been fixed.

Comment (kayw-geek) [reply lib/v2/learnku/topic.js]: Thank you very much for reviewing for me. I hope you can review it again sometime. @TonyRL

**Comment by kayw-geek** [comment]
@DIYgod

File: lib/v2/learnku/topic.js [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+            return {
+                title,
+                description: art(path.join(__dirname, 'templates/topic.art'), {
+                    article,
+                    comment,
+                }),
+                category: categoryName,
+                link: itemLink,
+                pubDate: parseDate($('.timeago').attr('title'), 'YYYY/MM/DD'),
+            };
```
Comment (TonyRL) [inline lib/v2/learnku/topic.js:52]: Can you cache the whole item instead of the web page?

**Pull request merged**